### DISCUSSION TITLE
Can Isar replace shared preferences

### DISCUSSION THREAD

**Discussion by eb-so** [isar/isar#1555] [Help]
I don't want to have multiple solutions in my code base to store data
Why not make Isar also store simple data like booleans and strings?

Note: I don't know if that's possible or even recommended

**Comment by Cierra-Runis** [comment]
It's not good experience.
Isar is a database, it store a collection of class that has same structure.
I used to save `Config` to isar, but my experience is bad.
shared_persistence is necessary for storing key-value type of data, and it's convenience too.
